albumentations/imgaug/transforms.py

```diff
@@ -116,9 +151,26 @@ class IAAPiecewiseAffine(DualIAATransform):
         image, mask
     """
 
-    def __init__(self, scale=(0.03, 0.05), nb_rows=4, nb_cols=4, p=.5):
+    def __init__(self, scale=0, nb_rows=4, nb_cols=4, order=1, cval=0, mode="constant", p=.5):
```

Comment:
I think it would be better to use single quotes for `mode='constant'` because we mainly use single quotes in the other parts of the project.